src/RequestMaker/ParameterTable.js

```diff
@@ -59,13 +61,22 @@ const TableRowEditable = ({
         />
       </TableCell>
       <TableCell>
-        <TextField
-          isQuiet
-          value={keyItem}
-          width='100%'
-          placeholder='key'
-          onChange={(value) => onUpdate && onUpdate(index, { key: value })}
-        />
+        {type && type==='header' ? (
```

Comment:
Instead of checking the type let's always use `AutoHeader` for the `TextField` but only pass in defaultMenuItems for headers.